libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -45,9 +47,17 @@ const readingListReducer = createReducer(
       error: action.error
     };
   }),
+
+  // Action(addToReadingList) triggers the effects to call API to add book into reading list 
```

Comment:
Please add reducer implementation for `addToReadingList` & `removeFromReadingList` actions here.